raiden/network/proxies/token_network.py

```diff
@@ -1095,24 +1116,22 @@ def close_light(
         else:
             onchain_channel_identifier = channel_onchain_detail.channel_identifier
             if onchain_channel_identifier != channel_identifier:
-                msg = (
+                raise RaidenUnrecoverableError((
                     f"The provided channel identifier does not match the value "
                     f"on-chain at the provided block ({given_block_identifier}). "
                     f"This call should never have been attempted. "
                     f"provided_channel_identifier={channel_identifier}, "
                     f"onchain_channel_identifier={channel_onchain_detail.channel_identifier}"
-                )
-                raise RaidenUnrecoverableError(msg)
+                ))
 
             if channel_onchain_detail.state != ChannelState.OPENED:
-                msg = (
+                raise RaidenUnrecoverableError((
                     f"The channel was not open at the provided block "
                     f"({given_block_identifier}). This call should never have "
                     f"been attempted."
-                )
-                raise RaidenUnrecoverableError(msg)
+                ))
```

Comment:
good change, but it looks like there are redundant parentheses

Reply:
Yup i will remove them, thanks